frontend/test/components/AnnotationView/AnnotationViewHeader.spec.js

```diff
@@ -111,4 +113,31 @@ describe('AnnotationViewHeader.vue', () => {
 
     expect(wrapper.emitted('changed'), 'the "changed" event happened').to.be.undefined;
   });
+
+  it('should render third party links', () => {
+    const wrapper = getMountedComponent();
+
+    const mondayLink = wrapper.find('[data-test="monday-link"]');
+    expect(mondayLink.exists()).toBe(true);
```

Comment:
When verifying in unit tests is statements are true, make sure to keep the style of assertions to be the https://www.chaijs.com/api/bdd/ ChaiJS BDD instead style instead of Jest form.  Here is an example of https://www.chaijs.com/api/bdd/#method_true.

However, in this case, you can simplify the unit test with changing up the line on 120 and get rid of the true/false assertion completely.
https://test-utils.vuejs.org/api/#get Will throw an error if the returned DOM element doesn't exist.  This error will be more informative of what the failure is for the unit test instead of saying you got a true/false value.

Go through the unit tests in this PR and update all instances to use the ChaiJS BDD assertions.

Reply:
Whoops, yeah, that was an unexpected change that slipped past me with copilot. I'll go fix those things

Reply:
I am seeking clarification on the apparent discrepancy in addressing my use of ".toBe()" while its application by other team members and presence in the codebase seems unremarkable. I am committed to adhering to the ChaiJS BDD style and promoting consistency in our coding practices. My objective is to ensure the fair application of these standards for all team members, contributing to a cohesive development environment.

![image](https://user-images.githubusercontent.com/4248757/231221828-03490b4c-fca4-4c27-a228-4cfbe8779580.png)
![image](https://user-images.githubusercontent.com/4248757/231221937-3a4794c3-772d-4fed-821f-66e8e7d3bfac.png)
![image](https://user-images.githubusercontent.com/4248757/231222013-8d6d0720-0800-439d-a3dd-2004774f38cc.png)
![image](https://user-images.githubusercontent.com/4248757/231222135-0be3e824-69ea-4e4a-a704-1ec7ad47b03a.png)
![image](https://user-images.githubusercontent.com/4248757/231222211-5580eeef-2fba-4cd4-9536-6d4b8fd7bfc2.png)
![image](https://user-images.githubusercontent.com/4248757/231222299-9924c331-575a-4382-ac69-29af2844a2ae.png)

Reply:
During the months up until we reached our goal of developing the MVP we were all in a rush to develop as fast as we could and couldn't spend time being as thorough in development and in reviews.  We have been changing them slowly over time to make things as consistent as possible now that we are not developing within a time crunch.

Reply:
Since I have noticed these, do you want me to go ahead and switch them over so they are also following the ChaiJS BDD standard? I will be glad to do so.

Reply:
Only if they are in the files you are touching right now.  That is what I have been doing. They will eventually all get changed over.  Don't want a bunch of changes not related to this PR to get included.  Makes the commit history murkier.